### ISSUE TITLE
show my current location

### ISSUE THREAD

**Issue by DoronYosha** [CrocoBlock/suggestions#7090]
Hi
It would be great if you added the option to show my current location on a map, like
<img width="578" alt="צילום מסך 2023-09-14 ב-17 13 56" src="https://github.com/Crocoblock/suggestions/assets/42466275/4bfd0899-a0ee-401e-b7ca-5e1e46928fe7">

and the user will be able to walk and see its live location on the map for navigation.

Thank you
Doron

**Comment by lepsii** [comment]
Yes! I also need this for a customer! This blue dot that Google has in the map anyway. Should be easy to implement?
